bin/ttmscalr

```diff
@@ -1079,10 +1079,12 @@ Main {
 
     def run
       options = Scalr::Caller.collect_options(params, [:farm_id])
-      if params['server'].value.nil?
-        exec_role = fetch_role_for_script(options,'debug')
+      role = params['server'].value
+      role = 'debug' if role.nil?
```

Comment:
You could do `role = params['server'].value || 'debug'` here and make this one line if you want